.goreleaser.yaml

```diff
@@ -0,0 +1,46 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
```

Comment:
We might as well remove the generated comments. Since this file was generated, maybe replace the two lines with
```
# Generated by running `goreleaser` and then modified.
```

Reply:
Sure, I can strip all non-relevant comments and link to the upstream documentation